source/module_hamilt_pw/hamilt_stodft/sto_forces.cpp

```diff
@@ -106,29 +106,8 @@ void Sto_Forces::cal_stoforce(ModuleBase::matrix& force,
     }
 	
 	if(ModuleSymmetry::Symmetry::symm_flag == 1)
-	{
-		double *pos;
-		double d1,d2,d3;
-		pos = new double[GlobalC::ucell.nat*3];
-		ModuleBase::GlobalFunc::ZEROS(pos, GlobalC::ucell.nat*3);
-		int iat = 0;
-		for(int it = 0;it < GlobalC::ucell.ntype;it++)
-		{
-			//Atom* atom = &ucell.atoms[it];
-			for(int ia =0;ia< GlobalC::ucell.atoms[it].na;ia++)
-			{
-				pos[3*iat  ] = GlobalC::ucell.atoms[it].taud[ia].x ;
-				pos[3*iat+1] = GlobalC::ucell.atoms[it].taud[ia].y ;
-				pos[3*iat+2] = GlobalC::ucell.atoms[it].taud[ia].z;
-				for(int k=0; k<3; ++k)
-				{
-                    p_symm->check_translation(pos[iat * 3 + k], -floor(pos[iat * 3 + k]));
-                    p_symm->check_boundary(pos[iat * 3 + k]);
-                }
-				iat++;				
-			}
-		}
-		
+    {
+        double d1, d2, d3;
```

Comment:
There are some code deleted without annotation, @Qianruipku can you review this code block?